lib/src/arithmetic/floating_point/floating_point_values/floating_point_value.dart

```diff
@@ -392,12 +449,13 @@ class FloatingPointValue implements Comparable<FloatingPointValue> {
 
   /// Convert a floating point number into a [FloatingPointValue]
   /// representation. This form performs NO ROUNDING.
-  factory FloatingPointValue.fromDoubleIter(double inDouble,
+  @internal
+  factory FloatingPointValue.ofDoubleUnrounded(double inDouble,
```

Comment:
The value add of this constructor is not clear given that `ofDouble` lets user to specify rounding mode.